packages/connect-common/files/coins.json

```diff
@@ -2201,7 +2201,13 @@
         {
             "blockchain_link": {
                 "type": "solana",
-                "url": ["https://solana.trezor.io"]
+                "url": [
+                    "https://solana.trezor.io",
```

Comment:
lets check with @Dehumanizer77 if we realy run `solana.trezor.io` it does not match pattern of other coins that always have a number suffix